api/build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         junitVersion = '5.3.2'
-        jacksonDataformatVersion = '2.9.10'
+        jacksonDataformatVersion = '2.10.0'
```

Comment:
I'm surprised this worked without changes to SpringFox, I got a NoClassDefFoundError when trying to run it with this version, did you change anything else? I notice the swagger is different...

Reply:
I updated Spring as well